### DISCUSSION TITLE
Building an exe

### DISCUSSION THREAD

**Discussion by DanJBower** [dotnet/maui#6631] [Q&A]
Is it possible to build a .exe from MAUI?

Currently, when I build an application targeting windows, it makes appears to make a windows app that it has to be installed.

Ideally the end goal would be to create a single executable file that can be shared without being installed. This is because my workplace has a range of restrictions on machines that makes it difficult to install normal applications and even more difficult to install windows apps, and more difficult still to install apps with self signed certificates. In the past I've made applications in WPF and have been able to make a single exe from that project. However, I've soon got to start a new project and would really like to transition to MAUI

**Comment by Hakim83** [comment]
As per [Microsoft documentation](https://docs.microsoft.com/en-us/dotnet/maui/windows/deployment/overview), for now that is not possible:
<blockquote>
.NET MAUI currently only allows publishing an MSIX package. You can't yet publish a Windows executable file for distribution.
</blockquote>

hopefully that will be supported soon.

**Comment by xtuzy** [comment]
i find a create exe video about winui3 https://youtu.be/CotNgFCnoj4

**Comment by deviprasad987** [comment]
@DanJBower Any update on creating executable file

**Comment by deviprasad987** [comment]
Please find the work around from below link
https://stackoverflow.com/questions/70097744/publish-net-maui-application-as-windows-executable

**Reply by dusrdev** [reply]
It doesn't work, at least not for everyone...
There's a comment on the answer that shows the error, and after trying I am getting that same error, and no one has posted a fix...
